core/app/c/(public)/[communitySlug]/public/forms/[formSlug]/fill/ExternalFormWrapper.tsx

```diff
@@ -151,15 +159,15 @@ export const ExternalFormWrapper = ({
 	const handleSubmit = useCallback(
 		async (
 			formValues: FieldValues,
-			evt: React.BaseSyntheticEvent<SubmitEvent> | undefined,
+			evt: React.BaseSyntheticEvent | undefined,
 			autoSave = false
 		) => {
 			const pubValues = preparePayload({
 				formElements,
 				formValues,
 				toggleContext,
 			});
-			const submitButtonId = evt?.nativeEvent.submitter?.id;
+			const submitButtonId = isSubmitEvent(evt) ? evt?.nativeEvent.submitter?.id : null;
```

Comment:
This is admittedly kind of annoying to have to do, but it's because `form.handleSubmit` only takes `React.BaseSyntheticEvent`, and again bc `strict: true` enables `strictFunctionTypes`, you cannot assign a more strict function type (using `evt: React.BaseSyntheticEvent<SubmitEvent>` where `evt: React.BaseSyntheticEvent` is expected).